helpers/mathjax.js

```diff
@@ -69,7 +69,7 @@ export class HtmlBlockMathRenderer {
 				elm.setAttribute('style', lineBreakStyle);
 			});
 			await window.MathJax.startup.promise;
-			window.MathJax.typeset([elem]);
+			await window.MathJax.typesetPromise([elem]);
```

Comment:
`typesetPromise` basically wraps the usual MathJax rendering functions in the same `handleRetriesFor` promise we're manually using before.